frontend/src/pages/Graph/GraphPage.tsx

```diff
@@ -228,7 +234,7 @@ const graphLegendStyle = kialiStyle({
   overflow: 'hidden'
 });
 
-const GraphErrorBoundaryFallback = () => {
+const GraphErrorBoundaryFallback = (): JSX.Element => {
```

Comment:
If it is possible, it is better to use React.ReactNode (more info here https://www.totaltypescript.com/jsx-element-vs-react-reactnode)

Reply:
Updated.